packages/client/src/utils/dateConvertors.ts

```diff
@@ -0,0 +1,11 @@
+export const convertISOtoTimeDateMonth = (date: string) => {
+  const targetDate = new Date(date)
+  const hours = targetDate.getHours()
+  const minutes = targetDate.getMinutes()
+  const month = targetDate.getMonth()
+  const day = targetDate.getDay()
+
+  return `${hours}:${minutes < 10 ? `0${minutes}` : minutes} ${
+    day < 10 ? `0${day}` : day
+  }.${month < 10 ? `0${month}` : month}`
```

Comment:
```suggestion
  return `${hours}:${minutes.padStart(2, '0')} ${day.padStart(2, '0')}.${month.padStart(2, '0')}`
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart